CalendarKit-Swift/SwiftCal/ICSEventParser.swift

```diff
@@ -399,6 +399,8 @@ class ICSEventParser: NSObject {
     }
     
     private static func attendee(from icsString: String) -> EventAttendee? {
+      var icsString = icsString
```

Comment:
This project doesn't follow our tab/whitespace indentation. Better to make this consistent with rest of project.